api/routes.py

```
@@ -0,0 +1,294 @@
+        + ("" if is_gemini_available else ", but Gemini service is unavailable"),
+    }
+
+
+@router.post(
+    "/food/analyze/text",
+    response_model=FoodAnalysisResult,
+    summary="Analyze food from text",
+    tags=["Food"],
+)
+async def analyze_food_by_text(
+    request: FoodAnalysisRequest, gemini: GeminiService = Depends(get_gemini_service)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)